app/enterprise/0.31-x/oidc-auth0.md

```diff
@@ -44,5 +44,5 @@ For basic authentication, use your client ID as the username and your client sec
 [client-credentials-grant]: https://auth0.com/docs/api-auth/tutorials/client-credentials
 [create-auth0-api]: https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0
 [non-interactive-client]: https://auth0.com/docs/clients
-[create-api]: /latest/admin-api/#add-api
+[create-api]: /0.12.x/admin-api/#add-api
```

Comment:
This section doesn't appear on the autogenerated netlify preview.

Reply:
https://docs.konghq.com/0.12.x/admin-api/#add-api